skyportal/tests/api/test_instrument.py

```diff
@@ -86,7 +86,7 @@ def test_token_user_post_get_instrument(super_admin_token):
         [
             d['field_id'] == 1
             and d['contour']['features'][0]['geometry']['coordinates'][0][0]
-            == [110.84784299030288, -87.01522509948724]
+            == [110.84791974982103, -87.01522999646508]
```

Comment:
I'm curious why the coordinates changed?

Reply:
I think some subtle things in the round trip.